core/utils/http.go

```diff
@@ -92,6 +92,7 @@ func withRetry(
 	}
 	for {
```

Comment:
I'll be removing the last reference to this function in my next PR so I think it's okay to simply ignore it for now.